doc/apiary/v2/fiware-ngsiv2-rc-2016_05_31.apib

```diff
@@ -1848,188 +1779,6 @@ Response:
 
 + Response 204
 
-# Group Registrations
-
```

Comment:
Release Candidate text shouldn't include explicit references to open dicussions. That references to open discussions are fine, but within the context of the WIP version of the speficiaiton (i.e. the one that will remain in "/latest").